tests/packagedcode/data/plugin/podspec-package-expected.json

```diff
@@ -0,0 +1,72 @@
+{
+  "headers": [
+    {
+      "tool_name": "scancode-toolkit",
+      "options": {
+        "input": "<path>",
+        "--json": "<file>",
+        "--package": true,
+        "--processes": "-1",
+        "--strip-root": true
+      },
+      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+      "message": null,
+      "errors": [],
+      "extra_data": {
+        "files_count": 1
+      }
+    }
+  ],
+  "files": [
+    {
+      "path": "SwiftLib.podspec",
+      "type": "file",
+      "packages": [
+        {
+          "type": "podspec",
+          "namespace": null,
+          "name": "SwiftLib",
+          "version": "0.0.1",
+          "qualifiers": {},
+          "subpath": null,
+          "primary_language": "Swift",
```

Comment:
I know I agreed to Swift as a primary language, but is this really the case?
do we have stats on what is the most common language for Cocoapods?
Can you find some? It could well be that Objective-C is more common?

Reply:
Found Objective-C as a most common language :)